pkgs/top-level/all-packages.nix

```
@@ -21476,7 +21476,9 @@ with pkgs;
 
   maxflow = callPackage ../development/libraries/maxflow { };
 
-  mbedtls = callPackage ../development/libraries/mbedtls { };
+  mbedtls_2 = callPackage ../development/libraries/mbedtls/2.nix { };
+  mbedtls_3 = callPackage ../development/libraries/mbedtls/3.nix { };
+  mbedtls = mbedtls_3;
```

```suggestion
  mbedtls = callPackage ../development/libraries/mbedtls/3.nix { };
```
so people don't opt into version 3 explicitly